runtime/compiler/control/HookedByTheJit.cpp

```diff
@@ -5022,6 +5022,14 @@ static void jitStateLogic(J9JITConfig * jitConfig, TR::CompilationInfo * compInf
 
 
    // Enable/disable GCR counting
+   // We will take into account the JIT startup state only if TR_LookAtJitStateForGCREnablement
+   // environment variable is defined, or if the JVM is not a JITServer client
+   static char *lookAtJitStateForGCREnablementEnvVar = feGetEnv("TR_LookAtJitStateForGCREnablement");
+   bool lookAtJitStateForGCREnablement = (lookAtJitStateForGCREnablementEnvVar != NULL)
+#if defined(J9VM_OPT_JITSERVER)
+                                         || (persistentInfo->getRemoteCompilationMode() != JITServer::CLIENT)
```

Comment:
Should this be
```suggestion
                                         || (persistentInfo->getRemoteCompilationMode() == JITServer::NONE)
```

Otherwise, won't this also apply to the server?

Reply:
Or is the point that this should only be a client change, and the server/nojitserver modes should be left alone?

Reply:
Yes, only the client should see the change.